### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by newyork-anthonyng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Tapable
 
-The tapable packages exposes many Hook classes, which can be used to create hooks for plugins.
+The tapable packages expose many Hook classes, which can be used to create hooks for plugins.
```
Comment (ooflorent) [inline README.md]:
```suggestion
The tapable package exposes many Hook classes, which can be used to create hooks for plugins.
```

Comment (newyork-anthonyng) [reply README.md]: Updated 👍

File: README.md [hunk 2]
```diff
@@ -31,7 +37,7 @@ class Car {
 	constructor() {
 		this.hooks = {
 			accelerate: new SyncHook(["newSpeed"]),
-			break: new SyncHook(),
```
Comment (buoyantair) [inline README.md]: Nice catch haha

**Comment by sokra** [comment]
Thanks

**Pull request merged**